src/app/modules/main/app_search/location_menu_sub_item.nim

```diff
@@ -1,14 +1,26 @@
-import json, strformat
+import json, strformat, sequtils, sugar
 import base_item
+import ../../shared_models/[color_hash_item, color_hash_model]
 
 export base_item
 
+type
+  ColorHashSegment* = tuple[len, colorIdx: int]
```

Comment:
You define this type here and in result_item. Maybe you can move it to color_hash_model ?

Reply:
Thanks, very good suggestion, applied :+1: